openpype/hosts/maya/plugins/publish/extract_playblast.py

```diff
@@ -25,6 +36,15 @@ class ExtractPlayblast(publish.Extractor):
     optional = True
     capture_preset = {}
 
+    def _capture(self, preset, override_viewport_options, instance):
+        self.log.info(
+            "Using preset:\n{}".format(
+                json.dumps(preset, sort_keys=True, indent=4)
+            )
+        )
+
+        return capture.capture(log=self.log, **preset)
```

Comment:
This doesn't need arguments `override_viewport_options` and `instance`

Reply:
Good point.